content/3_Silver/Two_Pointers.mdx

```diff
@@ -280,3 +280,72 @@ public class Books {
 ## Problems
 
 <Problems problems="ad" />
+
+## Quiz
+
+<Quiz>
+  <Quiz.Question>
+    What's the two pointers algorithm?
+    <Quiz.Answer correct>
+      Keeping track of two pointers to find a pair of indices which satisfy some condition.
+      <Quiz.Explanation>
+        Correct.
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      A way to answer range sum queries in $\mathcal{O}(1)$ time.
+      <Quiz.Explanation>
+        Incorrect. You might be thinking of [Prefix Sums](https://usaco.guide/silver/prefix-sums), another silver topic.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      An algorithm which splits an array into two halves to efficiently find an element satisfying a certain condition.
+      <Quiz.Explanation>
+        Incorrect.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    When would a solution not be possible?
+    <Quiz.Answer>
+      If the array isn't sorted.
+      <Quiz.Explanation>
+        Incorrect.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      If the pointers overlap.
```

Comment:
Pointers are points, so by overlap, do you mean at the same point? If so, it would still work if for instance, they both started at x = 0.

Reply:
oops mb, i was thinking of a "collision" type of two pointers, i clarified the question :D